### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by imdrt**
Update documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
...
+```shell
 npm install --save git+https://github.com/Asymmetrik/node-fhir-server-core.git
+#OR
+yarn add git+https://github.com/Asymmetrik/node-fhir-server-core.git
 ```
 
-Once installed, the server can be required in the application.
 
+
+2. File `node-fhir-server-core/src/runner.js` allows custom authorization server which should be incorporated separately. The server portion can be included as followed:
```
Comment (Robert-W) [inline README.md]: We probably shouldn't refer them to runner.js, that is not meant for anyone to use except maintainers of this library. It is just there so we can actually test what we are building. Also, mentioning they can add custom authorization but not linking any other documentation for how to do that is probably going to lead to more confusion.

File: README.md [hunk 2]
```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
...
-	if (server) {
-		server.logger.info(`FHIR Server it at http://localhost:${config.port}`);
-	}
+  let server = await fhirServerCore(config).catch(console.error);
+  if (server) {
+    server.logger.info(`FHIR Server it at http://localhost:${config.port}`);
+  }
 };
 
 main();
```
Comment (Robert-W) [inline README.md]: I can help update this snippet, it is no longer used in this way and the recent updates to core have changed some requirements.

File: README.md (hunk 1 above)
Comment (imdrt) [reply README.md]: Right. Not sure if others would know how to create their own runner / include server portion separately without looking at runner.js.

File: README.md [hunk 3]
```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
...
+
+Understanding of JavaScript [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) and familiarity of the FHIR specification would be helpful.
 
 ## Getting Started
-Install `node-fhir-server-core` as followed:
 
-```shell
-# For yarn users
-yarn add git+https://github.com/Asymmetrik/node-fhir-server-core.git
+1. Install `node-fhir-server-core` initial modules with either NPM or Yarn:
```
Comment (jonterrylee) [inline README.md]: let's pick a convention.

File: README.md [hunk 4]
```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
...
 
 
 <img src="https://www.asymmetrik.com/wp-content/uploads/2018/01/FHIR-Server-Architecture_Update.png" width="800">
 
 
 ## Prerequisites
-[Node.js](https://nodejs.org/en/) version later than 7.6, **but** not 8.5 (see [Attention](#attention)), and an understanding of [promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) usage are required.
 
-Familiarity of the FHIR specification and whatever database containing the resources will assist in querying data and returning them in a format conforming to specification.
+1. Install [Node.js](https://nodejs.org/en/) latest version (7.7+, **but** not 8.5, see [Attention](#attention)) and NPM included with Nodejs.
```
[the remark this answers is not in the document]
Comment (imdrt) [reply README.md]: ok

[the remark this answers is not in the document]
Comment (imdrt) [reply README.md]: Windows user need to install NPM or Yarn separately. NPM is currently included with Nodejs.

File: README.md [hunk 5]
```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
...
+   
+Confirm successful start with the following:
+```
+info: FHIR Server started successfully
+```
+
+
 ## Usage
-For a complete list of all configuration options, supported profiles, their setups, best practices, authentication, etc. please consult our [Wiki](https://github.com/Asymmetrik/node-fhir-server-core/wiki). More documentation and examples will be added over time.
+More configuration options, profiles, best practices, authentication are at [Asymmetrik Wiki](https://github.com/Asymmetrik/node-fhir-server-core/wiki).
```
Comment (jonterrylee) [inline README.md]: I like the old one better??

File: README.md (hunk 1 above)
Comment (Robert-W) [reply README.md]: We should link them to the mongo repo then, runner.js is not going to be setup for production and should not be used as a reference, it is just for running a dev server for us. We should also show them how to do these kinds of things in the wiki. Much rather have people read documentation versus read code.

File: README.md (hunk 5 above)
Comment (imdrt) [reply README.md]: ok

File: README.md [hunk 6]
```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
...
+info: FHIR Server started successfully
+```
+
+
 ## Usage
-For a complete list of all configuration options, supported profiles, their setups, best practices, authentication, etc. please consult our [Wiki](https://github.com/Asymmetrik/node-fhir-server-core/wiki). More documentation and examples will be added over time.
+More configuration options, profiles, best practices, authentication are at [Asymmetrik Wiki](https://github.com/Asymmetrik/node-fhir-server-core/wiki). 
+
+A developed implementation of this core server with the MongoDB database is at 
+[Asymmetrik/node-fhir-server-mongo](https://github.com/Asymmetrik/node-fhir-server-mongo).
```
Comment (jonterrylee) [inline README.md]: Let's say it's an example.

maybe something like "For a working Mongo example... refer to  [Asymmetrik/node-fhir-server-mongo](https://github.com/Asymmetrik/node-fhir-server-mongo)"

File: src/server/server.js [hunk 7]
```diff
@@ -10,78 +10,75 @@ const {
 
 class Server {
```
Comment (Robert-W) [inline src/server/server.js]: This whole file has been changed in the latest updates with updated comments, you'll need to pull the latest from develop into your local branch.

File: README.md (hunk 1 above)
Comment (imdrt) [reply README.md]: Ok

**Pull request closed without merging**